### DISCUSSION TITLE
Any plans on adding shadcn/ui?

### DISCUSSION THREAD

**Discussion by bleedingAyush** [expo-starter/expo-template#21] [Ideas]
Hey guys, 
Is it possible to add shadcn/ui in the app? Are you guys planning to release a new version with shadcn/ui?

**Comment by AchrafBn** [comment]
Hi,
yes, the template uses  [react-native-reusables](https://github.com/mrzachnugent/react-native-reusables) which includes the  components of shadcn/ui

**Comment by AchrafBn** [comment]
@bleedingAyush if you want to add more components into your project you need to use their CLI  for example :

`npx @react-native-reusables/cli@latest add accordion
`

you can read more about it here : [Docs](https://rnr-docs.vercel.app/components/accordion/)
